### PR TITLE
Delete Feed DropdownMenu Button

### PR DESCRIPTION
**Pull request by Benson-chou**
This branch adds the ability to delete a feed by long pressing on it and clicking "Delete". This will remove it from the database and feedList, so it will no longer show up in the application and will need to be re-added if the user would like to see it again.

### CODE REVIEW

**Review by hisbaan — APPROVED** [review]
This looks great! It's unfortunate that we had to move the cards into MainActivity (although it doesn't really pose a problem) as it makes things "less modular" so to speak. Hopefully, in the future, we can find some way to fix that (I have a couple of ideas in mind).

In the future, with other accessibility features, we should probably add a dialogue popup that asks the user to confirm as this is a semi-permanent action (they can re-add the feed, but it will be gone and all old articles that are no longer served will be gone forever).

File: app/src/main/java/monster/minions/binocularss/activities/MainActivity.kt [hunk 1]
```diff
@@ -179,46 +241,216 @@ class MainActivity : ComponentActivity() {
...
+                    Column(
+                        modifier = Modifier.width(200.dp)
+                    ) {
+                        feed.title?.let { title ->
+                            Text(text = title, fontWeight = FontWeight.SemiBold)
+                        }
+                        val items = listOf("Delete")
+                        // Convert pixel to dp
+                        val xDp = with(LocalDensity.current) { (offset.x - 15).toDp() }
+                        val yDp = with(LocalDensity.current) { (offset.y - 50).toDp() }
```
Comment (hisbaan) [inline app/src/main/java/monster/minions/binocularss/activities/MainActivity.kt:362]: I just realized, this manual offset (of -15 for x and -50 for y) should probably be done in dp as the dp values will not change from phone to phone, whereas the pixel values will.

Comment (hisbaan) [reply app/src/main/java/monster/minions/binocularss/activities/MainActivity.kt:362]: I'll go ahead and fix that really quickly

**Review by eamonma — APPROVED** [review]
Great documentation, perhaps try to remove commented code before committing.

**Comment by hisbaan** [comment]
> Great documentation, perhaps try to remove commented code before committing. 

Agreed. The part with "incorporated into feed card" can be removed. Good observation.

**Pull request merged**